charts/fluentd-elasticsearch/templates/daemonset.yaml

```diff
@@ -109,6 +121,16 @@ spec:
           readOnly: true
         - name: config-volume
           mountPath: /etc/fluent/config.d
+      {{- if .Values.awsSigningSidecar }}
+      - name: {{ include "fluentd-elasticsearch.fullname" . }}-aws-es-proxy
+        image: abutaha/aws-es-proxy:0.9
```

Comment:
Sorry, totally overlooked that. 
The image should be configured from the values file too.

Imho it should look like:

```
awsSigningSidecar:
  enabled: false
  image: 
    repository: abutaha/aws-es-proxy
    tag: 0.9
```

Reply:
This makes sense to me. I'll update shortly.

Reply:
Thanks :-)

Reply:
Resolved.